### ISSUE TITLE
How to disable a single rule from putout/putout preset ?

### ISSUE THREAD

**Issue by Shuunen** [EvgenyOrekhov/eslint-config-hardcore#749]
Hi there,

First thank to the contributors for this useful plugin <3

I have an issue, given the following code : 

```ts
const cats = await fetch('/api/cats')
if (!cats.length) console.log('no cats !')
```

I have the following issue `Use `.length === 0` when checking length is zero. eslint(unicorn/explicit-length-check)`

I agree with this suggestion and fix the code to : 

```ts
const cats = await fetch('/api/cats')
if (cats.length === 0) console.log('no cats !')
```

But now I have this lint issue from `putout/putout` : `Avoid zero in assertions (conditions/remove-zero)`

Theses rules seems to override each other, I would like to disable `conditions/remove-zero`

If I add an eslint disable : 

```ts
const cats = await fetch('/api/cats')
// eslint-disable-next-line conditions/remove-zero
if (cats.length === 0) console.log('no cats !')
```

I have this error : `Definition for rule 'conditions/remove-zero' was not found.`

Because this rule come from `putout/putout` I have to disable the whole putout preset : 

```ts
const cats = await fetch('/api/cats')
// eslint-disable-next-line putout/putout
if (cats.length === 0) console.log('no cats !')
```

Problems here in my opinion : 

1. rules seems to override each other
2. I'm forced to disable the whole `putout/putout` preset to get rid of one rule
3. impossible to disable a single rule from `putout/putout` preset

Cheers 😃

**Comment by Shuunen** [comment]
I had the idea to check putout project after submitting this issue, 

It's possible to configure putout : https://github.com/coderaiser/putout#-configuration

I added a `.putout.json` with the following content : 

```json
{
  "rules": {
    "conditions/remove-zero": "off"
  }
}
```

And this 'conditions/remove-zero' rule does not appears anymore, hope this will help others encountering this problem

**Comment by EvgenyOrekhov** [comment]
Thanks for reporting! Fixed in https://github.com/EvgenyOrekhov/eslint-config-hardcore/commit/d059d9ffd3f9dd1cac6fed28cfb3e1fed7237ea8, released in [v36.3.2](https://github.com/EvgenyOrekhov/eslint-config-hardcore/releases/tag/v36.3.2). No need for custom `.putout.json`.

**Comment by Shuunen** [comment]
Awesome, thanks
